api/resources_portal/models/material.py

```diff
@@ -69,15 +69,17 @@ class Meta:
     grants = models.ManyToManyField("Grant", through="GrantMaterialAssociation")
 
     organism = ArrayField(base_field=models.TextField(), blank=True, null=True)
-    contact_name = models.TextField(blank=True, null=True)
-    contact_email = models.TextField(blank=True, null=True)
     publication_title = models.TextField(blank=True, null=True)
     pre_print_doi = models.TextField(blank=True, null=True)
     pre_print_title = models.TextField(blank=True, null=True)
     citation = models.TextField(blank=True, null=True)
     additional_info = models.TextField(blank=True, null=True)
     embargo_date = models.DateField(blank=True, null=True)
 
+    @computed(models.BooleanField(null=False))
```

Comment:
It is my understanding that this is required if we need the value in the DB but I don't think that that is required. I could be mistaken though. i think we just need it on instances of the model